docs/dyndns/configure.rst

```diff
@@ -55,12 +55,43 @@ why that is the case, see :ref:`determine-ip-addresses`.) It is **not** possible
 to set up IPv4 and IPv6 by using both update servers in an alternating fashion.
 
 To update both your IPv4 and IPv6 address at the same time, most routers need
-to be configured with an update URL that provides both IP addresses. For
-Fritz!Box devices, for example, the URL reads:
-``https://update.dedyn.io/?myipv4=<ipaddr>&myipv6=<ip6addr>`` (Note that the
-placeholders in this URL must remain unchanged; your router will substitute
-them automatically. To find out the placeholder names for your router, please
-refer to the manual of your device.)
+to be configured with an update URL that provides both IP addresses via query string
+paramters, e.g. ``https://update.dedyn.io/?myipv4=1.2.3.4&myipv6=fd08::1234``, and 
+provide placeholders for the respective addresses. To find out the placeholder names 
+for your router, please refer to the manual of your device. 
+
+Example: Fritz!Box Devices
+--------------------------
+
+For Fritz!Box devices, for example, the respective URL reads:
+``https://update.dedyn.io/?myipv4=<ipaddr>&myipv6=<ip6addr>``.
+
+=============================   =====
+Field                           Entry
+=============================   =====
+DynDNS Provider                 User-defined
+Update URL :superscript:`1`     ``https://update.dedyn.io/?myipv4=<ipaddr>&myipv6=<ip6addr>``
+Domain Name :superscript:`2`    mydomain.dedyn.io
+Username :superscript:`3`                        mydomain.dedyn.io
+Password :superscript:`4`                        52UJM9bfD6rrQlNWzPN6Xf482URQ
```

Comment:
Let's make it explicit that this is not the account password, but a token (we call it authorization token on the registration confirmation page). So, instead of `52UJM9bfD6rrQlNWzPN6Xf482URQ`, let's write something like `<your authorization token>`